src/main/java/com/suse/salt/netapi/datatypes/target/Glob.java

```diff
@@ -3,7 +3,7 @@
 /**
  * Target for specifying minions by glob pattern.
  */
-public class Glob implements Target<String> {
+public class Glob implements Target<String>, SshTarget<String> {
```

Comment:
`SshTarget` already extends `Target`, but `Target` is also included here just for readability. Otherwise it may look like Glob is *exclusively* used for salt-ssh.